[2023-09-27T21:08:54.798-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my-first-dag.extract_transform_load manual__2023-09-28T01:08:50.246857+00:00 [queued]>
[2023-09-27T21:08:54.803-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my-first-dag.extract_transform_load manual__2023-09-28T01:08:50.246857+00:00 [queued]>
[2023-09-27T21:08:54.804-0400] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2023-09-27T21:08:54.906-0400] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): extract_transform_load> on 2023-09-28 01:08:50.246857+00:00
[2023-09-27T21:08:54.910-0400] {standard_task_runner.py:57} INFO - Started process 98260 to run task
[2023-09-27T21:08:54.912-0400] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'my-first-dag', 'extract_transform_load', 'manual__2023-09-28T01:08:50.246857+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmprrch4k9m']
[2023-09-27T21:08:54.913-0400] {standard_task_runner.py:85} INFO - Job 43: Subtask extract_transform_load
[2023-09-27T21:08:55.026-0400] {task_command.py:415} INFO - Running <TaskInstance: my-first-dag.extract_transform_load manual__2023-09-28T01:08:50.246857+00:00 [running]> on host erosennnin-TUF-Gaming-FX505DU-FX505DU
[2023-09-27T21:08:55.278-0400] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ramesh@somemail.com' AIRFLOW_CTX_DAG_OWNER='Ramesh Sannareddy' AIRFLOW_CTX_DAG_ID='my-first-dag' AIRFLOW_CTX_TASK_ID='extract_transform_load' AIRFLOW_CTX_EXECUTION_DATE='2023-09-28T01:08:50.246857+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-28T01:08:50.246857+00:00'
[2023-09-27T21:08:55.279-0400] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-27T21:08:55.279-0400] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/erosennnin/Programing/Apache_Airflow/Create_and_execute_a_Shell_script_from_Airflow/my_first_dag.sh ']
[2023-09-27T21:08:55.288-0400] {subprocess.py:86} INFO - Output:
[2023-09-27T21:08:55.289-0400] {subprocess.py:93} INFO - /usr/bin/bash: line 1: /home/erosennnin/Programing/Apache_Airflow/Create_and_execute_a_Shell_script_from_Airflow/my_first_dag.sh: Permission denied
[2023-09-27T21:08:55.289-0400] {subprocess.py:97} INFO - Command exited with return code 126
[2023-09-27T21:08:55.294-0400] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/erosennnin/anaconda3/envs/coursera/lib/python3.11/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 126.
[2023-09-27T21:08:55.296-0400] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=my-first-dag, task_id=extract_transform_load, execution_date=20230928T010850, start_date=20230928T010854, end_date=20230928T010855
[2023-09-27T21:08:55.414-0400] {standard_task_runner.py:104} ERROR - Failed to execute job 43 for task extract_transform_load (Bash command failed. The command returned a non-zero exit code 126.; 98260)
[2023-09-27T21:08:55.444-0400] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-27T21:08:55.453-0400] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
